config/prow/issue_tracker_config.go

```diff
@@ -69,6 +69,15 @@ func generateIssueTrackerPeriodicJobs() {
 		daysToRot:   rotDefault,
 		daysToClose: closeDefault,
 	}.generateJobs()
+
+	// generate for knative/client-contrib
```

Comment:
Do you really need it now for this new repo? It's not enabled for `knative/client` yet 😆

Reply:
ok, we can skip that for now (and I will discuss with the team whether we want to enable the stale bot for all client stuff)